install_config/install/docker_registry.adoc

```diff
@@ -78,6 +78,17 @@ $ oadm registry --help
 ----
 endif::[]
 
+=== Deploying the Registry as a DaemonSet
+
+You may use the `oadm registry` command to deploy the registry as a DaemonSet with
```

Comment:
It would be nice to have the DeamonSet explained somewhere in the doc and linked here, just a thought :wink: